irohad/consensus/yac/storage/yac_storage_cleanup_strategy_impl.hpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_YAC_STORAGE_CLEANUP_STRATEGY_IMPL_HPP
+#define IROHA_YAC_STORAGE_CLEANUP_STRATEGY_IMPL_HPP
+
+#include "consensus/yac/storage/yac_storage_cleanup_strategy.hpp"
+
+#include <queue>
+
+namespace iroha {
+  namespace consensus {
+    namespace yac {
+      class BufferedCleanupStrategy : public CleanupStrategy {
+       public:
+        using QueueSizeType = size_t;
+        using RoundType = Round;
```

Comment:
Why do you need this? You use both `RoundType` in 23 line and raw `Rounds` in 27